src/dropin-model.js

```diff
@@ -215,7 +215,11 @@ function isPaymentOptionEnabled(paymentOption, options) {
     return gatewayConfiguration.paypalEnabled && Boolean(options.merchantConfiguration.paypalCredit);
   } else if (paymentOption === 'applePay') {
     applePayEnabled = gatewayConfiguration.applePayWeb && Boolean(options.merchantConfiguration.applePay);
-    applePayBrowserSupported = global.ApplePaySession && global.ApplePaySession.canMakePayments();
+    try {
```

Comment:
Could we instead do:

```js
applePayBrowserSupported = global.ApplePaySession && global.location.protocol === 'https:' && global.ApplePaySession.canMakePayments();
```

Reply:
Sure. I've just tested it and it also works.